//using Kafka.Ksql.Linq.Core.Exceptions;
using System;
using System.Collections.Generic;
using Xunit;

namespace Kafka.Ksql.Linq.Tests.Core;

//public class CoreExceptionsTests
//{
//    private class TestException : CoreException
//    {
//        public TestException(string m) : base(m) { }
//        public TestException(string m, Exception i) : base(m, i) { }
//    }

//    //[Fact]
//    //public void CoreConfigurationException_Constructors()
//    //{
//    //    var ex1 = new CoreConfigurationException("m1");
//    //    Assert.Equal("m1", ex1.Message);
//    //    var inner = new Exception("i");
//    //    var ex2 = new CoreConfigurationException("m2", inner);
//    //    Assert.Equal(inner, ex2.InnerException);
//    //}

//    [Fact]
//    public void CoreException_BaseConstructor_SetsMessage()
//    {
//        var ex = new TestException("msg");
//        Assert.Equal("msg", ex.Message);
//    }


//}
